enumexport error with call to s.Parse

When I try to run the enumexport tool like this 'go run ./generate/tools/enumexport.go', I saw this error:

```
# command-line-arguments
generate/tools/enumexport.go:158:17: not enough arguments in call to s.Parse
	have ()
	want (string)
generate/tools/enumexport.go:180:20: not enough arguments in call to typ.Parse
	have ()
	want (string)
```
I see the same error when I run it like this: 'go run ./generate/tools/enumexport.go appkit > ./generate/modules/enums/macOS/appkit'

My Info:
- Go version go1.20.1 darwin/arm64
- Mac OS 12.3.1
- ARM64 CPU

--- a/generate/codegen/gen_param.go
+++ b/generate/codegen/gen_param.go
@@ -35,7 +35,7 @@ func (p *Param) ObjcDeclare() string {
 
 func (p *Param) GoName() (name string) {
 	switch p.Name {
-	case "type", "range", "map", "string", "select":
+	case "type", "range", "map", "string", "select", "interface":
 		name = p.Name + "_"
 	default:
 		name = p.Name

--- a/generate/tools/enumexport.go
+++ b/generate/tools/enumexport.go
@@ -155,7 +155,7 @@ func exportConstants(db *generate.SymbolCache, framework *modules.Module, platfo
 			continue
 		}
 		if s.Kind == "Constant" && s.Type == "Global Variable" {
-			stmt, err := s.Parse()
+			stmt, err := s.Parse(platform)
 			if err != nil {
 				log.Fatalf("%s: %s in '%s'", s.Name, err, s.Declaration)
 			}
@@ -177,7 +177,7 @@ func exportConstants(db *generate.SymbolCache, framework *modules.Module, platfo
 				if typ.Declaration == "" {
 					return nil, false
 				}
-				typdef, err := typ.Parse()
+				typdef, err := typ.Parse(platform)
 				if err != nil {
 					log.Fatalf("%s: %s in '%s'", typ.Name, err, typ.Declaration)
 				}

